### PR TITLE
Added author details to latest post

### PR DESCRIPTION
**Pull request by metonymic-smokey**
@Gituser143 @tfidfwastaken I've added the author details and shortened some of the headings, since they were getting cut by the right nav pane.  

Is this okay and anything more to be done?

### CODE REVIEW

**Comment by tfidfwastaken** [comment]
You both need to add your author details in `_data/authors.yml` as per [this](https://tianqi.name/jekyll-TeXt-theme/docs/en/authors) format, and then I'll merge it.

File: _posts/2020-07-07-delving-into-docker.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+---
+ title: "Delving into Docker"
+ tags: docker containers
+ author: Bhargav SNV Aditi Ahuja
```
Comment (tfidfwastaken) [inline _posts/2020-07-07-delving-into-docker.md]: Also change this line to:
`Bhargav SNV and Aditi Ahuja`
Not sure if that^ will work, but this current one definitely won't.

**Comment by metonymic-smokey** [comment]
@tfidfwastaken done. Check it now.

**Comment by tfidfwastaken** [comment]
LGTM!

**Pull request merged**